Adds a way to create a custom popup template

SidFerreira opened the pull request:
At this moment we can't create custom popups.
If you want a popup without buttons, it will have a big gap under it.
If you want an image only, the title will keep some space too.

Those are only suggestions.
Adding a class on the popup would help too.

adamdbradley commented:
Would you be able to create some accomplaning unit tests to go with this? https://github.com/driftyco/ionic/blob/master/test/unit/angular/service/popup.unit.js

And why did you remove cssClass?

SidFerreira commented:
Added tests and fixed some code.
Important to notice: This change may break some layouts because if no buttons, title nor subtitle is set, the popup-head and popup-buttons will not be shown!

adamdbradley commented:
Sorry, but our next release is a release candidate and we're holding back new features or breaking changes, but rather fixes to the current feature set. Thank you for the PR, but this will have to wait. Thanks
